modules/mono/glue/Managed/Managed.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|x86' ">
     <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
+    <DebugType>portable</DebugType>
```

Comment:
Is this change meant to be part of this commit?

Reply:
No, we moved DebugType to portable for all csproj files, but it looks like I missed this one. Found it while making these fixes.